confluent_kafka/src/confluent_kafka.c

```diff
@@ -1584,8 +1588,10 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 
                 v = cfl_PyUnistr_AsUTF8(vs, &vs8);
 
-                if(rd_kafka_conf_set(conf, "plugin.library.paths", v, NULL, 0) != RD_KAFKA_CONF_OK) {
-                        cfl_PyErr_Format(RD_KAFKA_RESP_ERR__INVALID_ARG, "%s", NULL);
+                if (rd_kafka_conf_set(conf, "plugin.library.paths", v, NULL, 0)
+                   != RD_KAFKA_CONF_OK) {
+                        cfl_PyErr_Format(RD_KAFKA_RESP_ERR__INVALID_ARG,
+                                "%s", NULL);
```

Comment:
"%s", NULL will raise an exception with the message "(null)".
Instead pass errstr to conf_set, and then use that here.

Reply:
Yep sorry that was left over from when I did the initial POC